pkg/server/router/operation.go

```diff
@@ -125,21 +138,38 @@ func (pdr OperationRouter) GetOperations(ctx context.Context, w http.ResponseWri
 			util.LoggingErrorMsg(err, "invalid get operations request"), http.StatusBadRequest)
 	}
 
-	ops, err := pdr.service.GetOperations(req)
+	ops, err := o.service.GetOperations(req)
 	if err != nil {
 		logrus.WithError(err).Error("getting operations from service")
 		return framework.NewRequestError(err, http.StatusInternalServerError)
 	}
-	resp := GetOperationsResponse{Operations: make([]Operation, len(ops.Operations))}
-	for i, op := range ops.Operations {
-		resp.Operations[i].ID = op.ID
-		resp.Operations[i].Done = op.Done
-		resp.Operations[i].Result.Error = op.Result.Error
-		resp.Operations[i].Result.Response = op.Result.Response
+	resp := GetOperationsResponse{Operations: make([]Operation, 0, len(ops.Operations))}
+	for _, op := range ops.Operations {
+		resp.Operations = append(resp.Operations, routerModel(op))
 	}
 	return framework.Respond(ctx, w, resp, http.StatusOK)
 }
 
+func routerModel(op operation.Operation) Operation {
+	return Operation{
+		ID:   op.ID,
+		Done: op.Done,
+		Result: OperationResult{
+			Error:    op.Result.Error,
+			Response: serviceToRouterType(op.Result.Response),
+		},
+	}
+}
+
+func serviceToRouterType(result interface{}) any {
+	switch result.(type) {
+	case model.Submission:
```

Comment:
what's the point of this method?

also haven't seen the any type - how is it different than `interface{}`?